templates/bin_setup_review_app.erb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
```

Comment:
> Unless we intend to support Solaris 8,  this should be
>
> #!/bin/sh

All other scripts are using `env`:

```
./bin/rake:#!/usr/bin/env ruby
./bin/rspec:#!/usr/bin/env ruby
./bin/setup:#!/usr/bin/env sh
./bin/suspenders:#!/usr/bin/env ruby
./spec/fakes/bin/heroku:#!/usr/bin/env ruby
./spec/fakes/bin/hub:#!/usr/bin/env ruby
./templates/bin_setup:#!/usr/bin/env sh
./templates/bin_setup_review_app.erb:#!/usr/bin/env sh
```

Should we change them as part of a different PR?